scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -371,7 +371,18 @@ task ExtractTask {
         --filter-set-name ~{filter_set_name}'
     fi
 
-    gatk --java-options "-Xmx~{memory_gib - 3}g" \
+    # This tool may get invoked with "Retry with more memory" with a different amount of memory than specified in
+    # the input `memory_gib`, so use the memory-related environment variables rather than the `memory_gib` input.
+    # https://support.terra.bio/hc/en-us/articles/4403215299355-Out-of-Memory-Retry
+    if [[ ${MEM_UNIT} == "GB" ]]
+    then
+        memory_mb=$(python3 -c "from math import floor; print(floor((${MEM_SIZE} - 3) * 1000))")
+    else
+        echo "Unexpected memory unit: ${MEM_UNIT}" 1>&2
+        exit 1
+    fi
+
+    gatk --java-options "-Xmx${memory_mb}m" \
```

Comment:
oooooh i didn't know we could do this!
(can we do this all over the VAT pipeline too!??!?!)

Reply:
Sure why not. The hard part is remembering to tick the 'Retry with more memory' option when launching the workflow lol

Reply:
And we already need to remember to click on the reference disk checkbox for that...